src/Umbraco.Web.UI.Client/src/less/components/editor/umb-editor.less

```diff
@@ -17,13 +17,14 @@
 }
 
 .umb-editor--animating {
-    will-change: transform, opacity, width, left;
+    //will-change: transform, opacity, width, left;
```

Comment:
No need to keep commented out code like this, that's what source control is for :) There's a couple of these in this PR

Reply:
Sorry, my mistake. I where suppose to get them out of there.